src/plugins/cordova-plugin-vibration/app-host.js

```diff
@@ -87,8 +87,9 @@ module.exports = function (messages) {
 
         for (x = 1; x <= times; x++) {
             body.animate({ left: -10 }, 5,
-                function () { checkForVibration(deferred, body);
-            })
+                function () {
```

Comment:
```
body.animate({ left: -10 }, 5, function () {
    checkForVibration(deferred, body);
})
.animate(...)
...
```

maybe? It seems like everywhere else we've typically put `function () {` on the same line as the initial call